packages/unplugin-typia/tests/fixtures/__snapshots__/vite/random.js

```
@@ -39,8 +1,120 @@
+var _randomFormatEmail = {};
+var _randomString = {};
+var _randomInteger = {};
+var hasRequired_randomInteger;
+function require_randomInteger() {
+  if (hasRequired_randomInteger) return _randomInteger;
+  hasRequired_randomInteger = 1;
+  Object.defineProperty(_randomInteger, "__esModule", { value: true });
+  _randomInteger._randomInteger = void 0;
+  const _randomInteger$1 = (props) => {
+    var _a, _b, _c, _d;
+    let minimum = (_a = props.minimum) !== null && _a !== void 0 ? _a : ((_b = props.multipleOf) !== null && _b !== void 0 ? _b : 1) * (props.maximum === void 0 ? 0 : props.maximum - 100);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)